test/unit/compute_resources/ovirt_test.rb

```diff
@@ -24,4 +24,29 @@ class OvirtTest < ActiveSupport::TestCase
       assert_find_by_uuid_raises(ActiveRecord::RecordNotFound, cr)
     end
   end
+
+  describe "associating operating system" do
+    setup do
+      @operating_systems_xml = Nokogiri::XML(File.read('test/fixtures/ovirt_operating_systems.xml'))
+      @compute_resource = FactoryGirl.build(:ovirt_cr).tap do |compute_resource|
+        fog_client = mock
+        ovirt_client = OVIRT::Client.new("", "", "")
+        ovirt_client.stubs(:http_get).returns(@operating_systems_xml)
+        fog_client.stubs(:client).returns(ovirt_client)
+        compute_resource.stubs(:client).returns(fog_client)
+      end
+      skip "Not supported version of rbovirt" unless @compute_resource.supports_operating_systems?
```

Comment:
Under what circumstance would I have Foreman with this patch but not an rbovirt new enough to do this?